cli/init.js

```diff
@@ -33,6 +33,7 @@ module.exports = async function init(args) {
     // You can also provide the "--no-save" option, and then "args.save"
     // will equal to false.
     else if (args.save == null) {
+      // eslint-disable-next-line no-console
```

Comment:
Nit: feels like you could minimize the diff by doing ESLint rule changes separately.